fdbserver/TLogServer.actor.cpp

```diff
@@ -2884,6 +2884,15 @@ ACTOR Future<Void> restorePersistentState(TLogData* self,
 		removed.push_back(errorOr(logData->removed));
 		logsByVersion.emplace_back(ver, id1);
 
+		TraceEvent ev("Role", recruited.id());
+		ev.detail("As", Role::TRANSACTION_LOG.roleName)
+		    .detail("Transition", "BeginWithEpochInfo")
+		    .detail("Origination", "Restored")
+		    .detail("OnWorker", self->workerID)
+		    .detail("RecoveryCount", std::to_string(logData->recoveryCount))
+		    .detail("SharedTLog", self->dbgid.shortString());
+		ev.trackLatest(recruited.id().shortString() + ".Role");
```

Comment:
This traceevent looks a lot like the one below. Do we need two of them?

Reply:
You mean "TLogPersistentStateRestore"? I think I should merge my event and TLogPersistentStateRestore event.